KebPOSTests/ValidationTest.cs

```diff
@@ -70,4 +78,29 @@ public void IsValidAnswer_ShouldReturnFalse(string input)
 
         Assert.IsFalse(result);
     }
+
+    [Test]
```

Comment:
Tests that involve the database fall into the realm of integration tests. I'll create tickets for those so we don't have to test everything manually soon. For now just test the new validation methods you need to create.